app/jobs/ims_export_job.rb

```diff
@@ -0,0 +1,39 @@
+class ImsExportJob < ApplicationJob
+  queue_as :ims_export
+
+  retry_on StandardError, attempts: 16
```

Comment:
If an error occurs do we get some kind of notification? Maybe via rollbar?

Reply:
No, but we can choose to later if needed.